ui/webui/resources/BUILD.gn

```diff
@@ -185,6 +185,7 @@ leo_icons = [
   "brave-icon-outline.svg",
   "brave-icon-release-color.svg",
   "brave-icon-search-color.svg",
+  "brave-icon-search.svg",
```

Comment:
This icon ended up not being used... but I was thinking it might be a good one for WDP.

@bbondy do we want to add WDP to the settings page also?
This was recently added with https://github.com/brave/brave-core/pull/30023